### PR TITLE
chore: Nuget packages update

### PR DESCRIPTION
**Pull request by maximilien-noal**

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<Project>
+	<!-- Common packages -->
```
Comment (JorisVanEijden) [inline src/Directory.Build.props:2]: I don't think these are all required by this project.

File: src/Bufdio.Spice86/Bufdio.Spice86.csproj [hunk 2]
```diff
@@ -1,56 +1,21 @@
...
-		<version>7.0.0</version>
-		<Authors>Luthfi Tri Atmaja, Kevin Ferrare, Maximilien Noal, Joris van Eijden, Artjom Vejsel</Authors>
-		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-		<Description>Reverse engineer and rewrite real mode dos programs</Description>
-		<PackageProjectUrl>https://github.com/OpenRakis/Spice86</PackageProjectUrl>
-		<RepositoryUrl>https://github.com/OpenRakis/Spice86</RepositoryUrl>
-		<RepositoryType>git</RepositoryType>
+		<TargetFramework>net8.0</TargetFramework>
+		<Nullable>enable</Nullable>
+		<ImplicitUsings>enable</ImplicitUsings>
```
Comment (JorisVanEijden) [inline src/Bufdio.Spice86/Bufdio.Spice86.csproj:6]: This means some of the usings in the source files can be removed

File: src/Directory.Build.props (hunk 1 above)
Comment (maximilien-noal) [reply src/Directory.Build.props:2]: No they are not. But since dependabot is still broken, at least that way it will only mess up one file.

Comment (JorisVanEijden) [reply src/Directory.Build.props:2]: ah, the GitHub interface made it look like this file was in the Bufdio.Spice86 directory.

But I don't feel this is the right way to solve issues with dependabot.
All of the projects now have unused dependencies...

We can look into Dependabot alternatives (at work I use Renovate). Or we can just manually click the "Upgrade packages in solution" button in Rider once in a while.

Comment (maximilien-noal) [reply src/Directory.Build.props:2]: I'll put non common packages werr they belong so we can move on with this PR. 😊

Yeah replacing dependabot sounds good.

Comment (JorisVanEijden) [reply src/Directory.Build.props:2]: https://github.com/apps/renovate

Comment (maximilien-noal) [reply src/Directory.Build.props]: I re-did the PR, now Directory.Build.props only contains common packages (Sourcelink, source code analyzers)

File: src/Bufdio.Spice86/Bufdio.Spice86.csproj (hunk 2 above)
Comment (maximilien-noal) [reply src/Bufdio.Spice86/Bufdio.Spice86.csproj]: Done.

**Review by JorisVanEijden — APPROVED** [review]

**Pull request merged**